test/build/50_ch-grow.bats

```diff
@@ -44,7 +67,7 @@ setup () {
     [[ $status -eq 0 ]]
     [[ $output = /* ]]                                      # absolute path
     [[ $CH_GROW_STORAGE && $output = "$CH_GROW_STORAGE" ]]  # match what we set
-}
+}	
```

Comment:
This is a common gotcha — you'll want to double check diffs for extra whitespace creeping in. You can probably configure your editor to highlight it if you like.